examples/advanced/flutter/.packages

```diff
@@ -0,0 +1,29 @@
+# Generated by pub on 2019-11-28 11:10:50.302722.
```

Comment:
I think this file is generated by flutter. I'm not 100% sure, but I think it should not be included. You might want to double check that everything runs ok without it.

Further info on what to commit/not commit here:
https://dart.dev/guides/libraries/private-files

Reply:
The .packages is generated on pub get, so it's safe to delete